### PR TITLE
fix(condo): Doma-8869 fix permission check to access meters

### PR DESCRIPTION
**Pull request by dkoviazin**
Removed checks that resident unitName matches meter's unitName. Only checks for accountNumber matching

### CODE REVIEW

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/meter/schema/Meter.test.js [hunk 1]
```diff
@@ -847,7 +847,7 @@ describe('Meter', () => {
                 expect(meters).toHaveLength(0)
             })
 
-            test('resident: cannot read Meters in other unit in same property', async () => {
+            test('resident: can read Meters in other unit in same property if accountNumber is correct', async () => {
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/schema/Meter.test.js:850]: Is there a test for case, where accountNumber is not correct?

**Comment by pahaz** [comment]
Could you please add at least one negative test

File: apps/condo/domains/meter/schema/Meter.test.js [hunk 2]
```diff
@@ -860,24 +860,25 @@ describe('Meter', () => {
...
-                    accountNumber: billingAccount2.number,
+                    accountNumber: billingAccount1.number,
                     unitName: unitName2,
                 })
-                const meters = await Meter.getAll(client1, { id: meter.id })
-
-                expect(meters).toHaveLength(0)
+                const meters1 = await Meter.getAll(client1, { id: meter.id })
+                const meters2 = await Meter.getAll(client2, { id: meter.id })
+                expect(meters1.find(({ id }) => id === meter.id)).toBeTruthy()
```
[the remark this answers is not in the document]
Comment (dkoviazin) [reply apps/condo/domains/meter/schema/Meter.test.js]: Done

File: apps/condo/domains/meter/schema/Meter.test.js (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/domains/meter/schema/Meter.test.js:850]: Done

**Pull request merged**